util/include/json_output.hpp

```diff
@@ -86,42 +86,46 @@ class BuildJson {
   inline BuildJson() {}
 
   // Start a root JSON object.
-  inline void startJson() { str += "\n{\n"; }
+  inline void startJson() { str += "{\n"; }
```

Comment:
nit: I think I commented before, why do you need the inline keyword
Iso C++:
`A function defined within a class definition is an inline function.`

Reply:
Yes, you're right.  I will fix that and other review comments in an upcoming PR.